src/sys/unix/net.rs

```diff
@@ -54,6 +54,7 @@ pub(crate) fn new_socket(domain: libc::c_int, socket_type: libc::c_int) -> io::R
         target_os = "watchos",
         target_os = "espidf",
         target_os = "vita",
+        target_os = "nto",
```

Comment:
QNX supports the SOCK_CLOEXEC option, just not the SOCK_NONBLOCK option when creating a socket or a socketpair. https://www.qnx.com/developers/docs/7.1/#com.qnx.doc.neutrino.lib_ref/topic/s/socket.html.  

Like the code for uds/mod.rs, you can enable line 27 and SOCK_NONBLOCK should be ignored:
  let socket_type = socket_type | libc::SOCK_NONBLOCK | libc::SOCK_CLOEXEC;
  
Below, you can ignore the call to fcntl with FD_CLOEXEC on the socket (add qnx to the ignore list with espidf and vita) as it's already set on socket creation.